README.md

```diff
@@ -25,33 +34,74 @@ poetry show --tree
 poetry update
 ```
 
-Once you've added a dependency you will need to rebuild your container using:
+If developing within a container, you will need to rebuild after adding any new dependencies.
 
 ```bash
 docker-compose build
 ```
 
 ### Pre-commit
+A pre-commit configuration is included in the repo to assist in the development workflow.
+It's not required for development, but can be useful.
 
-Ensure that you have installed [pre-commit](https://pre-commit.com/#install) on your machine and that you have instantiated pre-commit in the **Blueprint API** repository by running `pre-commit install` in the root of your local copy of the repo.
+To use pre-commit, ensure that you have installed [pre-commit](https://pre-commit.com/#install) on your machine and that
+you have instantiated pre-commit in the **Blueprint API** repository by running `pre-commit install` in the root of your
+local copy of the repo.
 
-Once you have _pre-commit_ installed _pre-commit hooks_ will run each time that you do a git commit. _Pre-commit_ will try to resolve many issues, but you will be required to resolve those that it cannot before pushing your code.
+Once you have ``pre-commit`` installed, _pre-commit hooks_ will run each time that you do a git commit. ``pre-commit``
+will try to resolve many issues, but you will be required to resolve those that it cannot before pushing your code.
 
-### Testing
+### Create an admin user and using the admin site
+When developing locally, it's useful to have an admin user.
+To add a super-user to your local database, run:
+```bash
+python3 manage.py createsuperuser
+```
+And enter a username and password when prompted. 
+This will allow access to access the admin site, http://localhost:8000/admin/ where the database can be interacted with 
+directly.
+
+### Migrations
+When there are changes to the database models, migrations will need to be executed: 
```

Comment:
Should we add in that migrations will also need to be created via `python3 manage.py makemigrations` since we're adding in the migrate command too?

Reply:
I wasn't sure since my goal was just to make sure that someone could look at the readme and (hopefully) get things up and running.